### PR TITLE
Fix Requirements to match dataprofiler - scipy>=1.10.0

### PR DESCRIPTION
**Pull request by rxm7706**
Closes
https://github.com/capitalone/synthetic-data/issues/280
https://github.com/capitalone/synthetic-data/issues/340

Synced requirements between dataprofiler and synthetic-data
Rationale for choosing dataprofiler>=0.10.3 was based on trying to sync on scipy>=1.10.0
https://github.com/capitalone/DataProfiler/blob/0.10.3/requirements.txt#L13
dataprofiler 0.10.3 - increased compatibility to scipy>=1.10.0 ; 

Please review and let me know if you require any changes. 
Thanks & regards.

### CODE REVIEW

**Comment by rxm7706** [comment]
@taylorfturner I was not able to understand the error - Do you know what I missed here .
`pyupgrade................................................................Failed
- hook id: pyupgrade
- exit code: 1
- files were modified by this hook`

**Comment by taylorfturner** [comment]
@rxm7706 approved the checks run -- will take a look later in the day to ensure everything is looking good. Thanks for the contribution! CC @tazitoo

**Comment by rxm7706** [comment]
> checks are looking good -- will need an update branch #343 merged @rxm7706. Thanks!

@taylorfturner Thank you ! please review - updated branch.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-numpy>=1.22
+numpy>=1.22.0
 scikit-learn>=1.1.0
-scipy==1.8.0
-dataprofiler>=0.8.8
+scipy>=1.10.0
```
Comment (taylorfturner) [inline requirements.txt:3]: looks good to me -- were these causing particular issues?

File: setup.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def parse_requirements(filename):
     return [line for line in lineiter if line and not line.startswith("#")]
 
 
-with open("README.md") as readme:
+with open("README.md", encoding="utf8") as readme:
```
Comment (taylorfturner) [inline setup.py:28]: was the absence causing an error?

Comment (rxm7706) [reply setup.py:28]: see https://github.com/capitalone/synthetic-data/issues/280 for error

**Comment by rxm7706** [comment]
Hi @taylorfturner is this stuck -  or did I miss something here, anything for me to do. Thank you.

**Comment by taylorfturner** [comment]
oh thanks for the ping @rxm7706. oddly the mend license check isn't passing... I'd recommend an empty commit `--allow-empty` to trigger a check re-run and we'll see if that resolves. Thanks!

**Comment by rxm7706** [comment]
> oh thanks for the ping @rxm7706. oddly the mend license check isn't passing... I'd recommend an empty commit `--allow-empty` to trigger a check re-run and we'll see if that resolves. Thanks!

Done - Waiting Approval ! @taylorfturner

**Review by tazitoo — APPROVED** [review]
LGTM - thanks for the PR.

**Comment by taylorfturner** [comment]
> > oh thanks for the ping @rxm7706. oddly the mend license check isn't passing... I'd recommend an empty commit `--allow-empty` to trigger a check re-run and we'll see if that resolves. Thanks!
> 
> 
> 
> Done - Waiting Approval ! @taylorfturner 

If it doesn't pass the check, we can do some more research as to the cause.

**Comment by taylorfturner** [comment]
reopening to re-trigger checks

**Pull request merged**